scripts/regression_test/extension_regression_test.yml

```diff
@@ -0,0 +1,191 @@
+          cd azure-cli-extensions
+
+          # git config
+          GITHUB_TOKEN=$(az keyvault secret show --vault-name kv-azuresdk --name azclibot-pat --query value -otsv)
+          git config --global user.email "AzPyCLI@microsoft.com"
+          git config --global user.name "Azure CLI Team"
+          git remote add azclibot https://azclibot:${GITHUB_TOKEN}@github.com/azclibot/azure-cli-extensions.git
+          git checkout -b regression_test_$(Build.BuildId)
+          git push --set-upstream azclibot regression_test_$(Build.BuildId)
+
+- job: CLIExtensionRegressionTest
+  displayName: CLI Extension Regression Test
+  dependsOn: PrepareExtensionRegressionTest
+  timeoutInMinutes: 0
+  strategy:
+    maxParallel: 12
+    matrix:
+      instance1:
+        Instance_idx: 1
+      instance2:
+        Instance_idx: 2
+      instance3:
+        Instance_idx: 3
+      instance4:
+        Instance_idx: 4
+      instance5:
+        Instance_idx: 5
+      instance6:
+        Instance_idx: 6
+      instance7:
+        Instance_idx: 7
+      instance8:
+        Instance_idx: 8
+      instance9:
+        Instance_idx: 9
+      instance10:
+        Instance_idx: 10
+      instance11:
+        Instance_idx: 11
+      instance12:
+        Instance_idx: 12
+  pool:
+    name: ${{ variables.ubuntu_pool }}
+  steps:
+  - task: UsePythonVersion@0
+    inputs:
+      versionSpec: '3.10'
+      displayName: "Use Python 3.10"
+  - task: AzureCLI@1
+    displayName: 'checkout cli and extension repo'
+    inputs:
+      azureSubscription: 'Azure CLI release'
+      scriptLocation: inlineScript
+      inlineScript: |
+        set -ev
+        pwd
+        git clone https://github.com/Azure/azure-cli-extensions.git
+        cd azure-cli-extensions
+
+        # git config
+        GITHUB_TOKEN=$(az keyvault secret show --vault-name kv-azuresdk --name azclibot-pat --query value -otsv)
+        git config --global user.email "AzPyCLI@microsoft.com"
+        git config --global user.name "Azure CLI Team"
+        git remote add azclibot https://azclibot:${GITHUB_TOKEN}@github.com/azclibot/azure-cli-extensions.git
+
+        git fetch azclibot
+        git checkout -b regression_test_$(Build.BuildId) azclibot/regression_test_$(Build.BuildId)
+  - template: ../../.azure-pipelines/templates/azdev_setup.yml
+    parameters:
+      CLIExtensionRepoPath: ./azure-cli-extensions
+  - bash: |
+      set -ev
+
+      source env/bin/activate
+      cd azure-cli-extensions
+
+      az login -u $(CLI_LIVE_TEST_ACCOUNT) -p "$(CLI_LIVE_TEST_PASSWORD)"
+      az account set -s 0b1f6471-1bf0-4dda-aec3-cb9272f09590
+
+      python ../scripts/ci/automation_full_test.py "12" "$(Instance_idx)" "latest" "" "True" "extension"
+    displayName: "Rerun tests"
+  - bash: |
+      publishErrorModules='false'
+      if [[ -f '/mnt/vss/.azdev/env_config/mnt/vss/_work/1/s/env/test_results_error_modules_$(Instance_idx).txt' ]]; then
```

Comment:
We may use variables instead of hard-coding the path `/mnt/vss` (https://github.com/Azure/azure-cli/pull/24201#discussion_r996689124).

Reply:
Per @wangzelin007 and my test, `$HOME`, `~` directs to `/home/cloudtest` instead of `/mnt/vss`

`Agent.HomeDirectory` -> `/usr/local/vss-agent/2.214.1`
`Pipeline.Workspace` -> `/mnt/vss/_work/1`

I can't find the exact variable for `/mnt/vss`☹️

Reply:
Or else we can use `$(Pipeline.Workspace)/../..`? Looks a liitle bit ugly to me

Reply:
☹️

Reply:
We change the `.azdev` folder from `/home/vsts/.azdev` to `/mnt/vss/.azdev` by mistake in #24201 

It should always be `~/.azdev`, which is `/home/cloudtest/.azdev` in new pipeline. I have changed all `.azdev` path back.